### DISCUSSION TITLE
[Logger] Outdated pino version

### DISCUSSION THREAD

**Discussion by louneskmt** [adonisjs/core#3792] [Help]
Hey! 

How frequently do you update the underlying `pino` package of `adonisjs/logger`? Looks like the version used is `v6.14.0` while the latest is `v8.1.0`, with some useful new features like `multistream`. `v6.X` LTS ended in April 2022.

Thanks.

**Comment by maxifuchs** [comment]
I would also be interested in a pino update due to the multistream option in newer versions

**Reply by thetutlage** [reply]
Have to double check the latest release of Pino and see if it can lead to a breaking change in `@adonisjs/core` or not. If it is a breaking change, then the Pino upgrade will take some time
